### PR TITLE
feat(point): add an intensity range for point clouds

### PR DESCRIPTION
**Pull request by zarov**
It is possible to specify the min and the max of the intensity range,
while updating the material in the same time to reflect those changes.

### CODE REVIEW

File: src/Layer/PointCloudLayer.js [hunk 1]
```diff
@@ -86,6 +93,12 @@ function markForDeletion(elt) {
  * `PointsMaterial`.
  * @property {number} [mode=MODE.COLOR] - The displaying mode of the points.
  * Values are specified in `PointsMaterial`.
+ * @property {number} [minIntensityRange=0] - The minimal intensity of the
+ * layer. Changing this value will affect the material, if it has the
+ * corresponding uniform.
+ * @property {number} [maxIntensityRange=65536] - The maximal intensity of the
```
Comment (gchoqueux) [inline src/Layer/PointCloudLayer.js]: It isn't intuitive the maximal intensity value.
`65536` is the `256x256` result , so the `intensity` is coded with a **`uint16`**. 
What it's the `intensity` specification in `Potree`, `las/laz` and `ept`?

Comment (zarov) [reply src/Layer/PointCloudLayer.js]: There are all in 16 bits, so that's why I choose this value.

Comment (gchoqueux) [reply src/Layer/PointCloudLayer.js]: Could you normalize the intensity to have values between 0.0 and 1.0?
It's more intuitive.

Comment (gchoqueux) [reply src/Layer/PointCloudLayer.js]: just normalize max and min

Comment (zarov) [reply src/Layer/PointCloudLayer.js]: Done

Comment (gchoqueux) [reply src/Layer/PointCloudLayer.js]: The material property could be normalized too, like Three js properties material.

Comment (gchoqueux) [reply src/Layer/PointCloudLayer.js]: and use [BufferAttribute.normalized](https://threejs.org/docs/index.html#api/en/core/BufferAttribute.normalized) for intensity Attribute.

Comment (zarov) [reply src/Layer/PointCloudLayer.js]: Done, I normalized it in LASParser, because this was the problem

Comment (gchoqueux) [reply src/Layer/PointCloudLayer.js]: but `bufferAttribute` isn't normalized.

Comment (zarov) [reply src/Layer/PointCloudLayer.js]: Sorry, I forgot to push the changes :sweat_smile:

**Comment by zarov** [comment]
PR updated @gchoqueux

**Review by gchoqueux — APPROVED** [review]

**Pull request merged**